projects/demo/src/modules/app/app.browser.module.ts

```diff
@@ -50,6 +51,7 @@ import {VersionManagerModule} from './version-manager/version-manager.module';
         VersionManagerModule,
         CustomHostModule,
         HttpClientModule,
+        MetrikaModule.forRoot({id: 87890624, webvisor: false, clickmap: false}),
```

Comment:
should we save it as environment variable ?

Reply:
id moved to env